Updated  "composer/installers" version to support custom path modules

miteshmap opened the pull request:
composer/installer latest tag is - v1.0.23 but updating it to master for this
https://github.com/composer/installers/pull/269

miteshmap committed:
Updated  "composer/installers" version to support custom path modules

composer/installer latest tag is - v1.0.23 but updating it to master for this
https://github.com/composer/installers/pull/269

mglaman commented on composer.json:
```diff
@@ -20,7 +20,7 @@
     ],
     "require": {
-        "composer/installers": "^1.0.20",
+        "composer/installers": "dev-master",
```
Could we append the current commit revision here, so it isn't always pulling `dev-master`?

`composer/installers: dev-master#f00o0oo0oo0oo` for example

miteshmap replied:
Yes, That's perfect.! I'll update it. :)

miteshmap committed:
added commit number.

https://github.com/composer/installers/commit/b3d1f3125a77af66d2bfd934c775150e4c9aecc3
